webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -238,7 +238,6 @@
           <a *ngIf="actionBar?.right?.sendTo"
             mmAuth="can_view_message_action"
             class="mm-icon mm-icon-inverse mm-icon-caption send-message desktop-only"
-            [attr.data-send-to]="actionBar?.right?.sendTo?._id"
```

Comment:
I searched but looks like it isn't used anywhere else, not even e2e. Taking feedback and removing it.